packages/@pollyjs/adapter-node-http/tests/integration/adapter-test.js

```diff
@@ -109,4 +113,30 @@ function commonTests(transport) {
 
     expect(nativeHash).to.equal(recordedHash);
   });
+
+  it('should be able to upload a file', async function() {
```

Comment:
I'm having a difficult time writing a test for this. Questions:

* Is there a way to run a _single_ test? Following [the recommendations](https://github.com/Netflix/pollyjs/blob/master/CONTRIBUTING.md#running-only-changed-tests) is very slow on my machine, and Testem's output is difficult to parse.
* Is this even the right location for a `multipart/form-data` test?
* How could I test the entire code path? It seems like `nativeRequest` just does some mocking?

Reply:
> Is there a way to run a single test? Following the recommendations is very slow on my machine, and Testem's output is difficult to parse.

I recommend the setup described under `"Running only changed tests"`.  This does not suffer from the same performance problems that `yarn test` has.

> Is this even the right location for a multipart/form-data test?

Since this test is specific to this adapter, I believe this is the correct place for it.

> How could I test the entire code path? It seems like nativeRequest just does some mocking?

If you want a more e2e/acceptance style test (which I think this may need) you can leverage the express server that runs along side when our tests run:
https://github.com/Netflix/pollyjs/blob/master/tests/middleware.js

Reply:
Thanks, I'll have a look at both testing options.